docs/api/browser-window.md

```diff
@@ -1049,6 +1049,17 @@ win.setSheetOffset(toolbarRect.height)
 
 #### `win.flashFrame(flag)`
 
+<!--
+```YAML history
+added:
+  - pr-url: https://github.com/electron/electron/pull/35658
```

Comment:
Don't think this is accurate here. That's the PR which created (well, exposed) `BaseWindow`, but `BrowserWindow.flashFrame` has existed for a long time, so it's inaccurate to document it as being added from that PR.

Reply:
Whoops, good catch! Removed in [`76da8b3` (#42982)](https://github.com/electron/electron/pull/42982/commits/76da8b346ddf429efc95e8709d5c7c01b5a9cf45)